js/gamestate.js

```diff
@@ -181,37 +203,37 @@ class GameState {
 
   // containerType can be board or hand
   selectCard(index, containerType, selectingPlayerName) {
-    let usePlayer
+    let actingPlayer
```

Comment:
aesthetically i like this renaming, it makes the variable make a lot more sense to me.